src/seedu/addressbook/commands/UpdateCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.person.Name;
+
+public class UpdateCommand extends Command {
+
+    public static final String COMMAND_WORD = "update";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Updates the person identified by the index number used in the last person listing.\n"
+            + "Parameters: INDEX\n"
+            + "Example: " + COMMAND_WORD + " 1";
```

Comment:
shouldn't this be:
```
"Example: COMMAND_WORD [index] [new name]"
```
the new name isn't reflected here.

Reply:
Added in!